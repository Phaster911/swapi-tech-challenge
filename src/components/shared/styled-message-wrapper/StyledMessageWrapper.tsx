import styled from "styled-components";

const StyledMessageWrapper = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  gap: 1rem;
  text-align: center;
`;

export default StyledMessageWrapper;
