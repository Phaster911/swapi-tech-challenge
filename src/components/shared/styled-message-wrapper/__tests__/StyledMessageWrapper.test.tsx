import { render } from "@testing-library/react";
import StyledMessageWrapper from "../StyledMessageWrapper";

describe("<StyledMessageWrapper />", () => {
  it("renders the component", () => {
    const { container } = render(<StyledMessageWrapper />);
    expect(container).toMatchSnapshot();
  });
});
