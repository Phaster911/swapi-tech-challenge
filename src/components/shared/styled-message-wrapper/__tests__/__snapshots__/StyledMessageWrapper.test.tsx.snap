// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<StyledMessageWrapper /> > renders the component 1`] = `
.c0 {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  gap: 1rem;
  text-align: center;
}

<div>
  <div
    class="c0"
  />
</div>
`;
